GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -69,7 +69,7 @@ public partial class FormPush : GitModuleForm
         private readonly TranslationString _pullRepository =
             new TranslationString("The push was rejected because the tip of your current branch is behind its remote counterpart. " +
                 "Merge the remote changes before pushing again.");
-        private readonly TranslationString _pullRepositoryButtons = new TranslationString("Pull with the default pull action ({0})|Pull with rebase|Pull with merge|Force push|Cancel");
+        private readonly TranslationString _pullRepositoryButtons = new TranslationString("Pull with the default pull action ({0})|Pull with rebase|Pull with merge|Force push with lease|Cancel");
```

Comment:
Have you checked that this button really triggers a `force push with lease`?
This PR only changes this label.

Reply:
Yes, it is doing a force with lease. If I haven't forgotten, I would have done this PR myself.